lang/node-homebridge/Makefile

```diff
@@ -0,0 +1,69 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NPM_NAME:=homebridge
+PKG_NAME:=node-$(PKG_NPM_NAME)
+PKG_VERSION:=0.4.46
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NPM_NAME)-$(PKG_VERSION).tgz
+PKG_SOURCE_URL:=http://registry.npmjs.org/$(PKG_NPM_NAME)/-/
+PKG_HASH:=c648a5a6f6bfb79b1d2eaac71d12ee2f6cd2221f33dacd5de2e6734004f19646
+
+PKG_BUILD_DEPENDS:=node/host
+PKG_USE_MIPS16:=0
```

Comment:
Does it actually fail with MIPS16 enabled or was it just copied from somewhere?  Has that been reported anywhere?  Is there a timeline for removing this workaround?

Reply:
@karlp 

First of all, node.js can not be built with MIPS16.
I decided not to use MIPS16 because I used to experience node modules that cause errors when built using MIPS16.